core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -349,4 +349,7 @@ private TableProperties() {}
 
   public static final String UPSERT_ENABLED = "write.upsert.enabled";
   public static final boolean UPSERT_ENABLED_DEFAULT = false;
+
+  public static final String AGGREGATE_PUSHDOWN_ENABLED = "aggregate.pushdown.enabled";
+  public static final String AGGREGATE_PUSHDOWN_ENABLED_DEFAULT = "false";
```

Comment:
If we have a nice fallback for any aggregation that cannot rely on the statistic completely, we can set the default to be `true`, or remove the property.

Reply:
Thanks for your comment!

I actually have thought about this when I wrote the code. The aggregate push down logic is decided inside `SparkScanBuilder`. I was debating myself if I should build the aggregates row inside `SparkScanBuilder` or `SparkLocalScan`. It seems more natural to build the aggregates row in `SparkLocalScan` so I put it there, but if I move this to `SparkScanBuilder`, then when I build the aggregates row using statistic, if the statistic are not available, I can fall back. I will change to that approach.